.gitignore

```diff
@@ -34,3 +34,5 @@ build/
 *-local.yml
```

Comment:
And one more thing, let's rename branch to `feature/#6_jobservice_docker_initialsetup` and use that pattern in the future as well.